pkg/distro/defs/rhel-9/distro.yaml

```diff
@@ -2073,7 +2086,9 @@ image_types:
         - *anaconda_pkgset
 
   gce:
-    name_aliases: ["gce-rhui"]
+    # this image type is set to `gcp` in image-builder-crc
+    # & `osbuild-composer`, so set the alias here
```

Comment:
This comment, IMHO,  does not provide much value, since the situation is exactly the same with other added aliases above, which don't have this comment.